compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -394,7 +394,10 @@
             { ParserRuleContext.OPEN_PARENTHESIS, ParserRuleContext.LT };
 
     private static final ParserRuleContext[] FUNCTION_KEYWORD_RHS =
-            { ParserRuleContext.FUNC_NAME, ParserRuleContext.OPEN_PARENTHESIS };
+            { ParserRuleContext.FUNC_NAME, ParserRuleContext.FUNC_TYPEDESC_OR_FUNC_KEYWORD_RHS};
```

Comment:
```suggestion
            { ParserRuleContext.FUNC_NAME, ParserRuleContext.FUNC_TYPEDESC_OR_FUNC_KEYWORD_RHS };
```

Reply:
Lets, rename `FUNC_TYPEDESC_OR_FUNC_KEYWORD_RHS` to `FUNC_TYPE_FUNC_KEYWORD_RHS`

Reply:
Fixed.